src/platform_impl/linux/x11/window.rs

```diff
@@ -117,6 +118,7 @@ pub struct UnownedWindow {
     root: xproto::Window,               // never changes
     #[allow(dead_code)]
     screen_id: i32, // never changes
+    sync_counter_id: u32,               // never changes
```

Comment:
This should be `Option<NonZeroU32>` to indicate that zero means an absence of this particular feature, or `NonZeroU32` to indicate that it will never be zero.

Reply:
Done! Made sense with querying the extension.